### DISCUSSION TITLE
Allow templating in contact fields like "to" in email

### DISCUSSION THREAD

**Discussion by killafunkinmofo** [grafana/grafana#52573] [Alerting]
Previously, grafana allowed contacts to simply be added to alerts. Alertmanager changes this quite a bit by each person required to make a notification policy and possibly a tag to an alert to help match. With 100s of people wanting to get just a couple alerts it seems to get kind of messy, and potentially problematic where a person may add a notification policy, not enable continue blocking anyone else from receiving the alert after.

I think in Alertmanager config, it allows you to use template string in the to: field for the receiving email addresses. https://prometheus.io/docs/alerting/latest/configuration/#email_config

I think this is possibly a clean way for individuals to subscribe to a few alerts without having to make notification policies or even a contact. A grafana instance could have a contact point called DynamicEmail with the to field set to a templated string like {{ .CommonLabels.dynamic_email }}. A single notification policy can be setup to match that dynamic_email tag. 

Is it possible to permit other fields of contacts to be templated string(like the to: field for email) to feature match prometheus alertmanager?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
